### PR TITLE
Pkg.s3

### PR DESCRIPTION
**Pull request by ned14**
https://github.com/cpp-pm/hunter/issues/230

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -81,17 +81,19 @@ matrix:
...
-        TOOLCHAIN=ios-nocodesign-11-4-dep-9-3
-        PROJECT_DIR=examples/s3
+# Disabled due to some weird breakage in curl dependency build
+#    - os: osx
+#      osx_image: xcode9.4
+#      env: >
+#        TOOLCHAIN=osx-10-13-cxx14
+#        PROJECT_DIR=examples/s3
+
+# Disabled due to log output exceeding maximum on travis
```
Comment (rbsheth) [inline .travis.yml]: https://hunter.readthedocs.io/en/latest/creating-new/create/cmake.html#fix-travis-log-too-long-error

Comment (ned14) [reply .travis.yml]: FYI now it exceeds the time limit instead: https://travis-ci.com/github/cpp-pm/hunter-testing/jobs/371441857

Comment (rbsheth) [reply .travis.yml]: :( ok

Comment (rbsheth) [reply .travis.yml]: Can you comment it from the travis yaml and put "Time limit exceeded" as a comment there?

**Pull request merged**